test/e2e/backup_test.go

```diff
@@ -0,0 +1,63 @@
+package e2e
+
+import (
+	"flag"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"golang.org/x/net/context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/vmware-tanzu/velero/pkg/cmd/cli/install"
+)
+
+var (
+	uuidgen              uuid.UUID
+	err                  error
+	backupName           string
+	restoreName          string
+	veleroInstallOptions *install.InstallOptions
+)
+
+func veleroInstall(cloudProvider string, useRestic bool) {
+	flag.Parse()
+	Expect(EnsureClusterExists(context.TODO())).To(Succeed(), "Failed to ensure kubernetes cluster exists")
+	Expect(err).To(Succeed(), "Failed to instantiate cluster client")
+	uuidgen, err = uuid.NewRandom()
+	Expect(err).To(Succeed())
+	veleroInstallOptions, err = GetProviderVeleroInstallOptions(cloudProvider, cloudCredentialsFile, bslBucket, bslPrefix, bslConfig, vslConfig, getProviderPlugins(cloudProvider))
+	Expect(err).To(Succeed(), "Failed to get Velero InstallOptions for provider azure")
+	veleroInstallOptions.UseRestic = useRestic
+	Expect(InstallVeleroServer(veleroInstallOptions)).To(Succeed(), "Failed to install Velero on kind cluster")
+}
+
+// Test backup and restore of Kibishi using restic
+var _ = Describe("Velero tests on Kind cluster using cloudProvider for object storage and Restic for volume backups", func() {
+	var (
+		client *kubernetes.Clientset
+		err    error
+	)
+	BeforeEach(func() {
+		veleroInstall(pluginProvider, true)
+		client, err = GetClusterClient()
+		Expect(err).To(Succeed())
```

Comment:
This check should be updated to include the previous message `"Failed to instantiate cluster client"`.

Reply:
Doh.. yes, removed.